js/src/button.js

```diff
@@ -86,7 +86,9 @@ class Button {
             input.checked = !this._element.classList.contains(CLASS_NAME_ACTIVE)
           }
 
-          $(input).trigger('change')
+          if (!avoidTriggerChange) {
+            $(input).trigger('change')
```

Comment:
I'm not a big fan of this parameter (`avoidTriggerChange`) I would prefer a class attribute.

For exemple here, when we trigger the change event and if it's an input element, we would set a class attribute to avoid to trigger too much events.

What do you think @Lausselloic ?

Reply:
Why not, I could add a `CLASS_NAME_CHANGING` to the button element instead of using a boolean parameter

Reply:
@Johann-S I let you check the update if it fit what you think

Reply:
Ok my bad too, I've updated the PR and create a class attribute shouldAvoidTriggerChange updated data value thru _jQueryInterface I think that's the idea